R/stat-qq-line.R

```diff
@@ -114,15 +119,13 @@ StatQqLine <- ggproto("StatQqLine", Stat,
    slope = diff(y_coords)/diff(x_coords)
    intercept = y_coords[1L] - slope * x_coords[1L]
 
-   if (length(line.expand) != 2) {
-     stop("Parameter line.expand must have length 2.", call = FALSE)
+   if(fullrange & !is.null(scales$x$dimension)){
```

Comment:
Needs space before `(`.

I think you should move the duplicated data frame creation code out of the `if` statement.